app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```diff
@@ -149,6 +149,7 @@ $after-button-padding-left: govuk-spacing(4);
 // Top level navigation links and search link, used when JavaScript is not available
 .gem-c-layout-super-navigation-header__navigation-item-link,
 .gem-c-layout-super-navigation-header__search-item-link {
+  color: govuk-colour("white");
```

Comment:
Nothing to fix here for now, but this style may not be needed, as it appears to be overridden by the following link styles:
https://github.com/alphagov/govuk_publishing_components/blob/main/app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss#L248-L257
```
...
@include focus-not-focus-visible {
  &,
  &:hover {
    text-decoration: none;
  }

  & {
    color: govuk-colour("white");
  }

  &:hover {
```